### ISSUE TITLE
Core dumps when attempting to use custom --seqence-error-model

### ISSUE THREAD

**Issue by iamh2o** [luntergroup/octopus#68]
**Describe the bug**
 I generated a new seqencing error model using the --data-profile flag and processing that file throuh the profiler.py script. When I pass the output file of the profiler script, I get a core dump right before alignment be

**Command**
Command line to run octopus:
'octopus --reference /mnt/disks/data5/data/reference/human/hs38DH/hs38DH.fa         --reads NA24385.aligned.sort.bam         -t regions.bed         --forest-file /mnt/disks/data5/data/reference/forests/forests/jem_na24385.wgs.forest         -o NA24385.octopus.vcf.gz         --threads 900         -X 2000MB         -B 50GB  --sequence-error-model ./PCR-free.INVITAE
`

**Desktop (please complete the following information):**
 - OS: AWS 96 core big mem machine
 - Version  v0.6.3-beta
 - Reference hg38

**Additional context**
This is the output of octopus:
`octopus --reference /mnt/disks/data5/data/reference/human/hs38DH/hs38DH.fa         --reads NA24385.aligned.sort.bam         -t regions.bed         --forest-file /mnt/disks/data5/data/reference/forests/forests/jem_na24385.wgs.forest         -o NA24385.octopus.vcf.gz         --threads 900         -X 2000MB         -B 50GB  --sequence-error-model ./PCR-free.INVITAE
[2019-06-04 00:08:00] <INFO> ------------------------------------------------------------------------
[2019-06-04 00:08:00] <INFO> octopus v0.6.3-beta (HEAD fd981298)
[2019-06-04 00:08:00] <INFO> Copyright (c) 2015-2019 University of Oxford
[2019-06-04 00:08:00] <INFO> ------------------------------------------------------------------------
[2019-06-04 00:08:04] <INFO> Done initialising calling components in 3s
[2019-06-04 00:08:04] <INFO> Detected 1 sample: "NA24385"
[2019-06-04 00:08:04] <INFO> Invoked calling model: individual
[2019-06-04 00:08:04] <INFO> Processing 3,088,286,401bp with 900 threads (128 cores detected)
[2019-06-04 00:08:04] <INFO> Writing filtered calls to "/mnt/disks/data5/results/OCT_TESTS/new_err_model_used/NA24385/NA24385.octopus.vcf.gz"
[2019-06-04 00:09:36] <INFO> ------------------------------------------------------------------------
[2019-06-04 00:09:36] <INFO>      current      |                   |     time      |     estimated   
[2019-06-04 00:09:36] <INFO>      position     |     completed     |     taken     |     ttc         
[2019-06-04 00:09:36] <INFO> ------------------------------------------------------------------------
Segmentation fault (core dumped)
`

This it the contents of my sequencing profile file:
`more PCR-free.INVITAE
A:45,45,41,41,40,37,35,33,29,25,23,21,20,20,19,18,17,17,16,16,15,15,14,14,13,12,11,11,10,8,8,8,7,6,6,6,6,6,5,4,4,4,4,4,4,4,4,4,4,4
T:45,45,41,41,40,37,35,33,29,25,23,21,20,20,19,18,17,17,16,16,15,15,14,14,13,12,11,11,10,8,8,8,7,6,6,6,6,6,5,4,4,4,4,4,4,4,4,4,4,4
C:45,45,41,38,36,32,28,22,19,16,15,13,11,10,7,7,6,5,5,5,4,4,4,4,4,4,4,4,4,3
G:45,45,41,38,36,32,28,22,19,16,15,13,11,10,7,7,6,5,5,5,4,4,4,4,4,4,4,4,4,3
NN:45,45,38,37,32,27,25,24,22,21,20,19,19,18,18,17,17,17,16,16,15,15,14,14,13,13,12,12,12,12,10,10,10,8,8,8,8,8,7,6,6,6,6,5,4,3,3,3,3,3
NNN:45,45,37,32,28,25,24,23,22,21,20,20,20,20,19,19,19,17,15,13,13,13,13,13,13,11,11,10,10,9,9,7,7,7,6,4,3
N:45,45,41,41,40,37,35,33,29,25,23,21,20,20,19,18,17,17,16,16,15,15,14,14,13,12,11,11,10,8,8,8,7,6,6,6,6,6,5,4,4,4,4,4,4,4,4,4,4,4`

**Comment by iamh2o** [comment]
When I switch back to --sequence-error-model PCR-free.NOVASEQ, things run fine.

**Comment by dancooke** [comment]
This was due to `profiler.py` not producing model parameters for tetra-nucleotide tandem repeats. I've added tetra and penta parameters in 1343216b3b9fe11175c2a34fed8d1abd833bd795 and 93a001acb5d8aed7e3a7b4fd78349f4df7975bce. I've also added a guard in the error model to avoid a bad access in-case the parameters are not specified (1343216b3b9fe11175c2a34fed8d1abd833bd795). You'll need to regenerate the error model with the updated `profiler.py` (no need to regenerate the data profile though).
